source/accessible-media/production-captions.html.erb.md

```diff
@@ -89,35 +94,58 @@ formatted and timestamped.
 
 ### Caption quality
 
-Always provide the highest-quality captions that convey 100% accuracy.
+{::nomarkup}
+<%= ref :start %>
+{:/nomarkup}
+
 See [resources for writing quality captions](#resources-for-writing-quality-captions) for more
-information. Keep the following basic points in mind when writing
+information.
+
+{::nomarkup}
+<%= ref :middle %>
+{:/nomarkup}
+
+Always provide the highest-quality captions that convey 100% accuracy.
```

Comment:
some human judgement is always necessary: if the cough is important within the context of the scene, include it as a sound-effect caption.  otherwise, no.